android/src/main/java/com/appgoalz/rnjwplayer/RNJWPlayerView.java

```diff
@@ -751,7 +751,9 @@ private void setupPlayer(ReadableMap prop) {
             }
 
             if (mColors.hasKey("timeslider")) {
-                CueMarkerSeekbar seekBar = findViewById(R.id.controlbar_seekbar);
+                // This line was crashing Android build on RN 0.73 upgrade
+                // CueMarkerSeekbar seekBar = findViewById(R.id.controlbar_seekbar);
+                CueMarkerSeekbar seekBar = findViewById(com.longtailvideo.jwplayer.R.id.controlbar_seekbar);
```

Comment:
After updating this, I was able to build the Android app and use the player. When viewing a video, I also was able to use the time slider on the player correctly